cmd/agent-js-worker/main.go

```diff
@@ -972,7 +972,7 @@ func prepareJWEEncrypter(opts *agentStartOpts, kmsStorageProvider storage.Provid
 		return nil, fmt.Errorf("failed to unmarshal JWE public key bytes to an EC public key object: %w", err)
 	}
 
-	jweEncrypter, err := jose.NewJWEEncrypt(jose.A256GCM, jose.DIDCommEncType, "", nil,
+	jweEncrypter, err := jose.NewJWEEncrypt(jose.A256GCM, "application/didcomm-encrypted+json", "application/didcomm-plain+json", "", nil, // nolint: lll
```

Comment:
@Baha-sk

Reply:
as discussed, this code should be refactored before going to prod @soluchok @DRK3 (ie move JWEEncrypter creation to edv) and use EDV media types instead of DIDComm's (cc @troyronda )